vocabularies/geological-observation-method.ttl

```diff
@@ -1,43 +1,461 @@
+    skos:inScheme <http://linked.data.gov.au/def/geological-observation-method> ;
+    skos:altLabel "Spectrometer"@en ;
+    skos:prefLabel "Spectrometry"@en .
+
+obsmh:spectroradiometer a skos:Concept ;
+    skos:broader obsmh:passive-spectral ;
+    skos:definition "A radiometer that measures the intensity of radiation in multiple wavelength bands (i.e., multispectral). Many times the bands are of high-spectral resolution, designed for remotely sensing specific geophysical parameters. Source:NASA EarthData"@en ;
+    skos:inScheme <http://linked.data.gov.au/def/geological-observation-method> ;
+    skos:altLabel "Spectroradiometer"@en;
+    skos:prefLabel "Spectroradiometry"@en .
+
+obsmh:interferometer a skos:Concept ;
+    skos:broader obsmh:passive-spectral ;
+    skos:definition "A family of techniques in which waves, usually electromagnetic waves, are superimposed, causing the phenomenon of interference, which is used to extract information."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/geological-observation-method> ;
+    skos:prefLabel "Interferometry"@en .
+
+obsmh:seismic-methods a skos:Collection ;
+    skos:definition "The classification of tools used as an energy source that generate a seismic signal for use in seismic survey acquisition."@en ;
+    skos:member obsmh:accelerated-weight-drop,
+        obsmh:air-gun,
+        obsmh:boomer,
+        obsmh:dynamite,
+        obsmh:firearm,
+        obsmh:hammer,
+        obsmh:hydraulic-oil-impactor,
+        obsmh:noise-source,
+        obsmh:plasma-sound-source,
+        obsmh:vibroseis,
+        obsmh:wacker,
+        obsmh:water-gun,
+        obsmh:weight-drop ;
+    skos:prefLabel "Seismic Methods"@en .
+
+obsmh:electrical-methods a skos:Collection ;
+    skos:definition "The classification of methods used in electrical geophysical survey acquisition."@en ;
+    skos:member obsmh:ip,
+        obsmh:sp,
+        obsmh:ert,
+        obsmh:ves,
+        obsmh:dcr,
+        obsmh:cr ;
+    skos:prefLabel "Electrical Geophysical Methods"@en .
+
+obsmh:em-methods a skos:Collection ;
+    skos:definition "The classification of methods used in electromagnetic geophysical survey acquisition."@en ;
+    skos:member obsmh:gpr,
+        obsmh:tem,
+        obsmh:sam,
+        obsmh:mlem,
+        obsmh:flem ;
+    skos:prefLabel "Electromagnetic Geophysical Methods"@en .
+    
+obsmh:mt-methods a skos:Collection ;
+    skos:definition "The classification of methods used in magnetotelluric geophysical survey acquisition."@en ;
+    skos:member obsmh:lpmt,
+        obsmh:csem,
+        obsmh:amt,
+        obsmh:bbmt ;
+    skos:prefLabel "Magnetotelluric Methods"@en .
+
+obsmh:spectral-methods a skos:Collection ;

```

Comment:
Here "Spectral" is used in place of "Remote Sensing", so I suggest renaming collection to Remote Sensing.  LIDAR RADAR etc are remote sensing methods but are not spectral methods. I question the need for a generic interferometer here also. Also, remove XRF from collection. The fact that these all utilise EMR as a stimulus or proxy for measuring various properties shouldn't be the basis for this collection.

Reply:
as in survey type above i would be ok with remote sensing

Reply:
Spectral links through to the spectral survey type. As per the other collections.
More accurately there should be a spatial collection and concepts should be moved into it, including synthetic aperture radar, radar, lidar, and alser altimetry.